config/webpack.config.js

```diff
@@ -1,141 +1,166 @@
 const path = require('path')
 
+const { sentryWebpackPlugin } = require('@sentry/webpack-plugin')
 const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const CopyPlugin = require('copy-webpack-plugin')
-const Dotenv = require('dotenv-webpack')
+const dotenv = require('dotenv')
+const DotenvPlugin = require('dotenv-webpack')
 const HTMLWebpackPlugin = require('html-webpack-plugin')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const { VueLoaderPlugin } = require('vue-loader')
 
+require('dotenv').config()
+
 function getPath (file) {
     return path.resolve(__dirname, '..', file)
 }
 
-module.exports = {
```

Comment:
This is much easier to see the changes (it's now wrapped in a function) using [?w=1](https://github.com/FlowFuse/flowfuse/pull/2862/files?w=1)